contrib/pg_tde/documentation/docs/how-to/uninstall.md

```diff
@@ -34,17 +37,51 @@ To uninstall `pg_tde`, follow these steps:
     !!! note
         The `DROP EXTENSION` command does not delete the underlying `pg_tde`-specific data files from disk.
 
-3. Run the `DROP EXTENSION` command against every database where you have enabled the `pg_tde` extension, if the goal is to completely remove the extension. This also includes the template databases, in case `pg_tde` was previously enabled there.
+## 2. Turn off WAL encryption
+
+If you are using  WAL encryption it needs to be turned off before the pg_tde library can be uninstalled.
+
+1. Run `ALTER SYSTEM SET pg_tde.wal_encrypt=off;`
 
-4. Remove any reference to `pg_tde` GUC variables from the PostgreSQL configuration file.
+2. Run `SELECT pg_reload_conf();` to reload server configuration
```

Comment:
It won't help. Changes of `pg_tde.wal_encrypt` require server restart. Until then server still will be writing encrypted WAL.